cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
This can be moved to some doc/md page

Reply:
I didn't get this. can you elaborate? 
```
/// <reference types="cypress" />
```
and this is to let file know about, this is cypress file so it provides relevant suggestions